web.py

```diff
@@ -79,27 +71,22 @@
         HOST_CACHE_SIZE cache entries.
     """
 
-    host_config = host_cache.get(hostname)
+    if hostname not in host_cache:
 
-    if not host_config:
+        config = {}
+
+        url = urljoin(FRANKLIN_API_URL, '/v1/domains/')
+        params = {'domain': hostname, 'key': FRANKLIN_API_KEY}
+        headers = {'User-Agent': 'franklin-server'}
```

Comment:
Ah, right `Authorization: Token {}` by default. I'll just add this now, change later if needed.